plugins/api/src/replica_close.cpp

```diff
@@ -380,23 +380,24 @@ namespace
 
             const auto is_write_operation = (O_RDONLY != (l1desc.dataObjInp->openFlags & O_ACCMODE));
 
+            const auto update_size = !json_input.contains("update_size") || json_input.at("update_size").get<bool>();
+            const auto update_status =
+                !json_input.contains("update_status") || json_input.at("update_status").get<bool>();
```

Comment:
Move this outside of the try-catch block so that the catch blocks can determine whether to update the status on error

Reply:
Done